From 3974f833f880aa971914b0696a0a0021f4d19f1e Mon Sep 17 00:00:00 2001
From: "Mao, Roger" <roger.mao@intel.com>
Date: Tue, 26 Dec 2023 13:32:21 +0000
Subject: [PATCH] Disable BT service in framework for blizzar project temporary
 Tracked-On: OAM-114394 Signed-off-by: maozhong1 roger.mao@intel.com

---
 services/java/com/android/server/SystemServer.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/java/com/android/server/SystemServer.java b/services/java/com/android/server/SystemServer.java
index f5f1d497878a..285d8189f995 100644
--- a/services/java/com/android/server/SystemServer.java
+++ b/services/java/com/android/server/SystemServer.java
@@ -1535,7 +1535,7 @@ public final class SystemServer implements Dumpable {
                 Slog.i(TAG, "No Bluetooth Service (Bluetooth Hardware Not Present)");
             } else {
                 t.traceBegin("StartBluetoothService");
-                mSystemServiceManager.startService(BluetoothService.class);
+                // mSystemServiceManager.startService(BluetoothService.class);
                 t.traceEnd();
             }
 
-- 
2.34.1

